index.html

```diff
@@ -0,0 +1,351 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<title>Youtube Page</title>
+	<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ"
+	 crossorigin="anonymous">
+	<link rel="stylesheet" type="text/css" href="./assets/stylesheet/style.css">
+
+</head>
+
+<body>
+	<header>
+		<div class="nav-bar">
+			<section class="header-left">
+				<i class="fas fa-bars"></i>
+				<img class="logo" src="./assets/media/images/youtube.png" alt="logo">
+			</section>
+			<section class="header-center">
+				<input type="text" class="search-field" placeholder="Search">
+				<button class="search-button">
+					<i class="fas fa-search"></i>
+				</button>
+			</section>
+			<section class="header-right">
+				<i class="fas fa-video"></i>
+				<i class="fas fa-th"></i>
+				<i class="fas fa-comment"></i>
+				<i class="fas fa-bell"></i>
+				<img class="user" src="./assets/media/images/user.png" alt="user">
+			</section>
+		</div>
+
+	</header>
+	<main>
+		<content>
```

Comment:
What HTML5 eelement is this? From my research, the one I found is deprecated https://developer.mozilla.org/en-US/docs/Web/HTML/Element/content